python/opentrons_ot3_firmware/constants.py

```diff
@@ -94,3 +96,15 @@ class ErrorCode(int, Enum):
     bad_checksum = 0x02
     invalid_byte_count = 0x03
     invalid_input = 0x04
+
+
+@unique
+class ToolType(int, Enum):
```

Comment:
How do we express that no tool is connected on a given carrier? We should have both the concept of "nothing is attached" and "something I don't know about is attached".

Reply:
threw in these other two values:
```
@unique
class ToolType(int, Enum):
    """Tool types detected on Head."""

    undefined_tool = 0x00
    pipette_96_chan = 0x01
    pipette_384_chan = 0x02
    pipette_single_chan = 0x03
    pipette_multi_chan = 0x04
    gripper = 0x05
    nothing_attached = 0x06
```